app/src/main/java/com/example/myapplication/SecondFragment2.java

```diff
@@ -133,7 +133,7 @@ public void afterTextChanged(Editable s) {
                     binding.textInput.setText(text);
                 }
                 binding.characterLimit.setText("Character Limit: " + binding.textInput.getText().length() + "/150");
-                MainActivity.teleOpNotes = binding.textInput.getText() + "";
+                UserModel.getMatchData().setNotes(String.format("%s", binding.textInput.getText()));
```

Comment:
Instead of format("%s", ...), I believe we can use `binding.textInput.getText().toString()`.